com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/GlobalLightLoopSettings.cs

```diff
@@ -218,9 +220,11 @@ internal static GlobalLightLoopSettings NewDefault() => new GlobalLightLoopSetti
         /// <summary>Maximum number of lights per ray tracing light cluster cell.</summary>
         public int maxLightsPerClusterCell;
 
-        /// <summary>Maximum size of one density volume texture.</summary>
-        public DensityVolumeResolution maxDensityVolumeSize;
-        /// <summary>Maximum number of density volumes at the same time on screen.</summary>
-        public int maxDensityVolumesOnScreen;
+        /// <summary>Maximum size of one Local Volumetric Fog texture.</summary>
+        public LocalVolumetricFogResolution maxLocalVolumetricFogSize;
+
+        /// <summary>Maximum number of Local Volumetric Fog at the same time on screen.</summary>
+        [Range(1, 512)]
```

Comment:
should the max value be  HDRenderPipeline.k_MaxVisibleLocalVolumetricFogCount instead?

Reply:
yeah thanks